### PR TITLE
feat(): Add sortSchema property

### PR DESCRIPTION
**Pull request by mattleff**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: https://github.com/nestjs/graphql/pull/997

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by kamilmysliwiec** [comment]
@johnbiundo would you be able to give it a quick review :)?

File: content/graphql/quick-start.md [hunk 1]
```diff
@@ -101,6 +101,15 @@ GraphQLModule.forRoot({
 }),
 ```
 
+By default, the types in the generated schema will be in the order they are defined in the included modules. To sort the schema lexicographically set the `sortSchema` property to `true`:
```
Comment (johnbiundo) [inline content/graphql/quick-start.md:104]:
```suggestion
By default, the types in the generated schema will be in the order they are defined in the included modules. To sort the schema lexicographically, set the `sortSchema` property to `true`:
```

Comment (johnbiundo) [reply content/graphql/quick-start.md:104]: Looks great!  One minor grammar suggestion.

**Comment by johnbiundo** [comment]
@kamilmysliwiec: review completed.  Looks great!

**Comment by kamilmysliwiec** [comment]
Thank you!

**Pull request merged**